crates/matrix-sdk/src/sliding_sync/list/mod.rs

```diff
@@ -930,38 +931,38 @@ mod tests {
         {
             let list = SlidingSyncList::builder("foo")
                 .sync_mode(SlidingSyncMode::Selective)
-                .ranges(ranges![(0, 1), (2, 3)].to_vec())
+                .ranges(vec![0u32..=1, 2..=3])
                 .build();
 
             {
                 let lock = list.inner.ranges.read().unwrap();
                 let ranges = Observable::get(&lock);
 
-                assert_eq!(ranges, &ranges![(0, 1), (2, 3)]);
+                assert_eq!(ranges, &vec![0..=1, 2..=3]);
             }
 
-            list.set_range(4u32, 5).unwrap();
+            list.set_range(4u32..=5).unwrap();
 
             {
                 let lock = list.inner.ranges.read().unwrap();
                 let ranges = Observable::get(&lock);
 
-                assert_eq!(ranges, &ranges![(4, 5)]);
+                assert_eq!(ranges, &vec![4..=5]);
             }
         }
 
         // Set range on `Growing`.
         {
             let list = SlidingSyncList::builder("foo").sync_mode(SlidingSyncMode::Growing).build();
 
-            assert!(list.set_range(4u32, 5).is_err());
+            assert!(list.set_range(4u32..=5).is_err());
```

Comment:
```suggestion
            assert!(list.set_range(4u64..=5).is_err());
```

I know `u32` would be accepted here because it implements `Into<u64>` or something like that, but I prefer the tests to be exemplary.